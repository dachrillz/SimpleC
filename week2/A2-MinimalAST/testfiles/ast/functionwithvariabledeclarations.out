Program
  List
    Function ID="a"
      List
        IdDecl ID="x"
        IdDecl ID="y"
