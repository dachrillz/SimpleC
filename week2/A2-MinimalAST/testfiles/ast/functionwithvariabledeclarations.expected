Program
  List
    Function ID="a"
      List
      Block
        List
          IdDecl ID="x"
          IdDecl ID="y"
